patches/minecraft/net/minecraft/world/item/crafting/BlastingRecipe.java.patch

```diff
@@ -0,0 +1,13 @@
+--- a/net/minecraft/world/item/crafting/BlastingRecipe.java
++++ b/net/minecraft/world/item/crafting/BlastingRecipe.java
+@@ -5,8 +_,8 @@
+ import net.minecraft.world.level.block.Blocks;
+ 
+ public class BlastingRecipe extends AbstractCookingRecipe {
+-   public BlastingRecipe(ResourceLocation p_43793_, String p_43794_, Ingredient p_43795_, ItemStack p_43796_, float p_43797_, int p_43798_) {
```

Comment:
you shouldn't remove old constructors, anyone could be using them

Reply:
Added new constructors alongside the old ones to take a category location.